cirq-google/cirq_google/ops/z_gate_family.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Union
+
+import cirq
+from cirq.ops import raw_types
+from cirq_google.ops.physical_z_tag import PhysicalZTag
+
+
+_PHYSICAL_Z_TAG = PhysicalZTag()
```

Comment:
What's the point of making this a constant?

Reply:
Was trying to save memory and runtime by not having to instantiate a new tag on every `__contains__` call, but actually instantiation time should be minimal and memory is freed once the program exits the function scope. So either way works - I could change it to instantiate a new tag every call.